templates/election-lookup.html

```diff
@@ -1,68 +1,70 @@
 {% extends 'layouts/main.html' %}
-
+{% import 'macros/cycle-select.html' as select %}
 {% block title %}
-  Election Lookup
+  Find candidates and elections by location
 {% endblock %}
 
 {% block body %}
-<div class="entity">
-  <header class="page-header slab slab--primary">
-    <h1 class="page-header__title">Elections</h1>
-    {{ search.search('page-header', 'candidates') }}
-  </header>
-  <section class="main">
-    <div class="container">
-      <h2>Find Elections by Location</h2>
-      <div id="election-lookup" class="row">
-        <div class="filters--horizontal">
-          <h4 class="filter__header">New Search</h4>
-          <form>
-            <div class="filter">
-              <label for="cycle" class="label">Election cycle</label>
-              <select id="cycle" name="cycle">
-                {% for cycle in cycles %}
-                <option value="{{ cycle }}">{{ cycle }}</option>
+<header class="page-header slab slab--primary">
+  <span class="page-header__title">Find candidates and elections by location</span>
+</header>
+<section class="main" id="election-lookup" >
+  <div class="container">
+    <h2 class="t-ruled--bottom t-ruled--bold">Search</h2>
+    <div class="search-controls">
+      <form>
+        <div class="search-controls__row">
+          <label class="label" for="cycle-select">Choose an election cycle</label>
+          {{ select.cycle_select(cycles, cycle) }}
```

Comment:
The cycle select widget changes the URL and refreshes the page on change, which we don't want to do here. I'll fix during review.